updater/lib/dependabot/setup.rb

```diff
@@ -50,6 +51,7 @@
 end
 
 Dependabot::OpenTelemetry.configure
+Dependabot::Sorbet::Runtime.silently_report_errors!
```

Comment:
`dependabot/setup` is only required in prod, so this wouldn't affect our test setup